.gitignore

```diff
@@ -1,31 +1,12 @@
 # generic (system) files/extensions we don't want
-*.seed
-*.log
-*.csv
-*.dat
-*.out
-*.pid
-*.gz
+node_modules
+lib
+coverage
+docs
+dist
+
 .idea/*
 *.iml
-*.DS_Store
-lib-cov
-pids
-logs
-results
 test-results.xml
-dist/docs/components
-dist/docs/font*
-dist/docs/grunt-styles
-dist/docs/img
-dist/docs/index.html
-coverage/
-
-/docs
-/dist
-node_modules
 npm-debug.log
-lib
-.bower
 
-.vscode
```

Comment:
If we're removing text editor/IDE configs, should we just go for all of them... ie. .idea, etc ? Also the comment about ```# generic (system) files/extensions we don't want``` could be wiped

Reply:
Sorry, didn't know what .vscode was, I'll add it back...and remove comment